csrankings-k.csv

```diff
@@ -40,6 +40,8 @@ Kai Chen 0005,HKUST,http://www.cse.ust.hk/~kaichen,NOSCHOLARPAGE
 Kai Chen 0012,Chinese Academy of Sciences,http://www.kaichen.org,i869JHoAAAAJ
 Kai Fischbach,University of Bamberg,https://www.uni-bamberg.de/sna/team/prof-dr-kai-fischbach,PRhJB5EAAAAJ
 Kai Fong Ernest Chong,SUTD,http://people.sutd.edu.sg/~ernest_chong,JewaBYEAAAAJ
+Kai H. Chang,Auburn University,http://www.eng.auburn.edu/~kchang,NOSCHOLARPAGE
+Kai Han 0001,University of Hong Kong,https://www.kaihan.org,tG8S_vMAAAAJ

```

Comment:
In Dept. of Statistics, not Computer Science. Please clarify how they meet inclusion criteria.

Reply:
Hi Emery, thanks for taking care of this.
Dr Han's research areas are in common CS areas: computer vision, machine learning and AI; and his PhD students are working on or will be working on these areas, with publications primarily in CVPR, ICCV, ECCV, NuerIPS, ICLR, etc.

Reply:
The inclusion criteria require that the faculty member be able to solely advise students for a PhD in Computer Science. Where they publish is immaterial.

Reply:
Of course, as a tenure-track Assistant Prof in Computer Vision and Machine Learning, he is able to and qualified to solely advise PhD students in Computer Science research areas, but not in the computer science department due to the department affiliation. He was an Assistant Prof at CS department of Bristol University before and did his PhD in CS department as well.

Reply:
Not in CS dept = this faculty member currently does not meet the criteria. Please remove this faculty member from the PR.